Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.5
+
+WORKDIR /usr/src/app
+
+RUN git clone https://github.com/digininja/CeWL/ 
+
+WORKDIR /usr/src/app/CeWL
+
+RUN gem install bundler
+
+RUN bundle install
+
+RUN gem source -c
+
+RUN gem install --user-install spider http_configuration mini_exiftool zip mime-types
```

Comment:
Why are you doing this after doing a bundle install?

Reply:
The bundle install command already performs the requirements installation right? Sorry, I don't dominate ruby.